test/Elastic.Apm.Tests/ApiTests/ApiTests.cs

```diff
@@ -16,34 +17,41 @@ namespace Elastic.Apm.Tests.ApiTests
 	/// </summary>
 	public class ApiTests
 	{
+		private const string TestSpan1 = "TestSpan1";
+		private const string TestSpan2 = "TestSpan1";
+		private const string TestTransaction = "TestTransaction";
+		private const string UnitTest = "UnitTest";
```

Comment:
Shouldn't it be `TestTransactionType = "UnitTest";` ?

Reply:
I thought that we can reuse it... In case we need it for e.g. `Span.Type`, or `Span.Name`, or `Transaction.Name`. Because in these tests none of those really matter.